server/graphql/v1/mutations/collectives.js

```diff
@@ -729,6 +729,14 @@ export async function deleteCollective(_, args, req) {
     throw new Error('Can not delete collective with paid expenses.');
   }
 
+  const eventCount = await models.Collectives.count({
+    where: { ParentCollectiveId: collective.id, type: types.EVENT },
```

Comment:
This is similarly broken.

Reply:
> This is similarly broken.

How to check whether it was fixed or not , how should I re run the build ??

Reply:
You should just test the feature manually. `models.Collectives` is undefined so this should not go far.

Reply:
> You should just test the feature manually. `models.Collectives` is undefined so this should not go far.

sorry about that, I missed it first time, since that function was not called so often and hence I thought may be it's ok, but `isdeletable` was called for almost everything and hence it was throwing error, I would be cautious next time.

Rectified the issue, made changes at both the places